pkg/processor/runtime/python/test/python_test.go

```diff
@@ -119,6 +121,66 @@ func (suite *TestSuite) TestAsyncHandler() {
 	}
 }
 
+func (suite *TestSuite) TestAsyncHandlerManyRequests() {
+	createFunctionOptions := suite.GetDeployOptionsAsync("asyncer",
+		suite.GetFunctionPath("outputter"))
+
+	createFunctionOptions.FunctionConfig.Spec.Handler = "async_outputter:handler"
```

Comment:
since you're using the `async_outputter`, I suggest also verifying you indeed wrote `numRequests` (100) lines in the file by sending a `async_read`  request after sending the requests.